app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    NotWearingMaskError.masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError as v_error:
+            return str(v_error)
```

Comment:
As per requirements:
`If at least one of the friend has problems with vaccines the function should return a message `"All friends should be vaccinated".`